src/text_render.rs

```diff
@@ -385,3 +553,29 @@ fn create_oversized_buffer(
 fn zero_depth(_: usize) -> f32 {
     0f32
 }
+
+#[cfg(feature = "custom-glyphs")]
+#[derive(Debug, Clone, Copy, PartialEq)]
+/// The input data to render a custom glyph
+pub struct CustomGlyphInput {
+    /// The unique identifier of the glyph.
+    pub id: crate::CustomGlyphID,
+    /// The size of the glyph in points (not scaled by the text area's scaling factor)
+    pub size: f32,
+    /// The scaling factor applied to the text area.
+    pub scale: f32,
+    /// Binning of fractional X offset
+    pub x_bin: cosmic_text::SubpixelBin,
+    /// Binning of fractional Y offset
+    pub y_bin: cosmic_text::SubpixelBin,
+}
+
+#[cfg(feature = "custom-glyphs")]
+#[derive(Debug, Clone)]
+/// The output of a rendered custom glyph
+pub struct CustomGlyphOutput {
```

Comment:
nit: `RasterizedCustomGlyph`

Reply:
What about just `RasterizedGlyph`?

Reply:
Actually, RasterizedCustomGlyph is fine.